### DISCUSSION TITLE
A Last Ben On Saturn Clowd

### DISCUSSION THREAD

**Discussion by simplejaky** [TheLastBen/fast-stable-diffusion#774] [General]
mv [Untitled.ipynb.zip](https://github.com/TheLastBen/fast-stable-diffusion/files/10127717/Untitled.ipynb.zip) Untitled.ipynb
Settings
![jak (13)](https://user-images.githubusercontent.com/119542302/204919324-dc692fbb-71be-49ad-abd8-e317c2e08444.png)
![jak (14)](https://user-images.githubusercontent.com/119542302/204919377-fe372493-2116-4000-9b73-99ca3e49a4d2.png)
![jak (15)](https://user-images.githubusercontent.com/119542302/204919404-d3c7a2d0-8e98-414c-8073-62159e074244.png)
so weel be
![jak (16)](https://user-images.githubusercontent.com/119542302/204919435-2a61a31d-be9b-4d7f-a1bf-0b34cdc4d89c.png)
if settings next..
![jak (17)](https://user-images.githubusercontent.com/119542302/204919527-34e9f038-d114-425b-8920-b6e2d486ceaa.png)
Fist Launch >>[upen some to 256mb fo download files]
![jak (18)](https://user-images.githubusercontent.com/119542302/204919562-ddb40f32-d485-4545-acca-ccff94764d61.png)
Now restart >>
![jak (19)](https://user-images.githubusercontent.com/119542302/204919685-0a9c3947-7cfe-4a51-a544-b7a386b7ec2c.png)
Fist this codes if skip big Bash >>[need up widgeds]
![jak (20)](https://user-images.githubusercontent.com/119542302/204919715-3d240ff8-e19e-4a25-ac3e-14ca9e571ea2.png)
Download StableDifusion15
![jak (21)](https://user-images.githubusercontent.com/119542302/204919746-faba29c1-346f-4eb9-9aef-03f0964d7ea2.png)
A.>>>so make a some png.. named(empye.png) put in workspace/dbmodels/stable-diffusion-v1-5
so copy on pc from
workspace/dbmodels/stable-diffusion-v1-5/scheduler/scheduler_config.json
now put upload scheduler_config.json in
B.>>>workspace/dbmodels/stable-diffusion-v1-5/HuggingFaceScheduler
now del some (clip blabla) from scheduler_config.json and(,)
put upload changed scheduler_config.json in
C.>>>workspace/dbmodels/stable-diffusion-v1-5/scheduler
dont call scheduler_config.json from JupyterLab like copy, sed etc : acc be blocked
Nou if skip big Bash >>
![jak (22)](https://user-images.githubusercontent.com/119542302/204919808-928ed712-4525-4ef4-83d3-9125caaba034.png)
Skip tools
![jak (23)](https://user-images.githubusercontent.com/119542302/204919839-141beb0d-54a6-4b7b-ad41-6332451b025a.png)
skeeps
![jak (24)](https://user-images.githubusercontent.com/119542302/204919868-90786467-9bc7-43e2-9ccf-298f3d20f0d4.png)
skeeps
![jak (25)](https://user-images.githubusercontent.com/119542302/204919907-c2432129-0b62-4ea0-bfa5-039996abddd8.png)
Now LastBenDiffusion
![jak (12)](https://user-images.githubusercontent.com/119542302/204919968-ff90f8c1-e49b-4993-8a9a-4aac8e2d4212.png)
so
![jak (27)](https://user-images.githubusercontent.com/119542302/204920085-8e665a9c-4bc6-408a-8e7c-b221f559e6c9.png)
stable-dif-15:		diffusion 1.5 with ckpts...
diffusers:		LastBen diffusers
input:			used fo dowloads main and class then gows in >>>
sessions:		name/files
worked:		fo pytorch sessions name/files
	: worked/onreused/ saves ckpt model
if need some file look in code
sesion>>>
![jak (28)](https://user-images.githubusercontent.com/119542302/204920113-816cdc69-d258-48b2-a12c-78a0f6835031.png)
sesion name launch
![jak (1)](https://user-images.githubusercontent.com/119542302/204920158-5927ff4c-4a2e-46cd-b9d4-407cb7bcda82.png)
files
![jak (2)](https://user-images.githubusercontent.com/119542302/204920183-d3e1d9a5-2c26-4ca1-ab90-34abeed0454d.png)
launch
![jak (3)](https://user-images.githubusercontent.com/119542302/204920223-a3058f3c-2821-4261-a497-21097a8bb03e.png)
less check than launch if need justs inicialize file
![jak (4)](https://user-images.githubusercontent.com/119542302/204920285-4c48b551-ec28-47b7-911c-f49686764f8b.png)
fist choise LBS: LastBenDif
HGF: HagginFase
![jak (5)](https://user-images.githubusercontent.com/119542302/204920348-7f1cf621-5ca0-4f63-92a6-2d1a84ebfe6c.png)
fp:		fp precission/no
gardient:	gardient
capion:		caption
wps:		with prior pevesasion
ens:		LBS encoding
Inst_words:	justs helps words
steps:		fo just
ens:		fo LBS encoding
pf:		per frame
foa:		fo all..
small window left of launs:	1-10 steps
![jak (6)](https://user-images.githubusercontent.com/119542302/204920395-175bb045-10ed-4a3f-805d-201643a758e8.png)
Gallilei
![jak (7)](https://user-images.githubusercontent.com/119542302/204920458-28f08601-bc77-42e8-8c3d-eefe042c4d71.png)
launch one time pos: positive nol: negative
Launch
![jak (8)](https://user-images.githubusercontent.com/119542302/204920484-b264130c-d348-451b-8641-6ea195dc1fab.png)
eptie seed: random else setuped
PT: session@PT eptie: PT last session s15@PT stable dif
Automatic
![jak (9)](https://user-images.githubusercontent.com/119542302/204920524-37234981-c677-4319-96a8-c80566cb960a.png)
Launch
![jak (10)](https://user-images.githubusercontent.com/119542302/204920559-b3b30de9-2f57-4251-8e64-7d6795022242.png)
Off
![jak (11)](https://user-images.githubusercontent.com/119542302/204920623-12b8d98b-cea4-4cf2-9a3f-5c488831ad26.png)
launch again 
![jak (26)](https://user-images.githubusercontent.com/119542302/204920656-61d89123-9cb4-4645-83b2-251a41bed51a.png)
than make cod a more petty
def()
if Untitled.ipynb==very well:
	uplod on geohab
cheers

**Comment by simplejakys** [comment]
[_l.py.zip](https://github.com/TheLastBen/fast-stable-diffusion/files/10151095/_l.py.zip)
[_s.py.zip](https://github.com/TheLastBen/fast-stable-diffusion/files/10151097/_s.py.zip)
placed in py so install deployments:

%run /home/jovyan/workspace/SCD/lights.py

up channel 256mb, up lab
then again lights
	>widgeds
	>xvels, lbs
makes png file and sheldue

%run /home/jovyan/workspace/SCD/sesion.py

time_info
auto_cachnamefiles/change in code if need excludes
bps>batch_size
gps>g_accumulation_steps
wu>wup

if textual_inv in automatic dont go
dont make veinv
fix: pip3 install torch==1.12.0 torchvision==0.13.0 torchaudio --extra-index-url https://download.pytorch.org/whl/cu113

**Comment by symplehonckis** [comment]
fix some errors
png files now generate automatics
[lights.py.zip](https://github.com/TheLastBen/fast-stable-diffusion/files/10163385/lights.py.zip)
[elisyd.py.zip](https://github.com/TheLastBen/fast-stable-diffusion/files/10163387/elisyd.py.zip)
[sesion.py.zip](https://github.com/TheLastBen/fast-stable-diffusion/files/10163388/sesion.py.zip)

**Comment by simplejackya** [comment]
somes  lights
[lights.py.zip](https://github.com/TheLastBen/fast-stable-diffusion/files/10262480/lights.py.zip)
aa  the same
-up 256 channel-
-hgf1.5-
-ipywidgets-
-xwhels-
-LBDdiffusion-
hgf can be choisen big-30GB ful download/n 5GB LBD download
if n/ downl. model_index.jsonw by fast-DreamBooth.ipynb
-in hgf1.5
 downl. hgf1.5/vae -256-512
-in hgf1.5/vae
sheuld
lates besesion

**Comment by simplejackyb** [comment]
[sesion.py.zip](https://github.com/TheLastBen/fast-stable-diffusion/files/10262553/sesion.py.zip)
set some sesion[without a "uwspaces"] somelike
![asd](https://user-images.githubusercontent.com/120786204/208515554-fa4cc5c7-1b20-45a7-8d75-8719ea2ad3bc.png)
choise instance
upload say oky
![asd (2)](https://user-images.githubusercontent.com/120786204/208515608-99d53e87-b592-4848-989a-0be03e5bf186.png)
download
![asd (3)](https://user-images.githubusercontent.com/120786204/208515632-734fa35a-cfda-4264-9aba-c05be6dfb448.png)
if split can be set time wait betwin
![2022_12_19_125](https://user-images.githubusercontent.com/120786204/208515805-4fe6b7af-3af7-4179-a166-e10b738c767f.png)
splits@timewait
![2022_12_19_125_11](https://user-images.githubusercontent.com/120786204/208516073-640ad463-e9c3-4472-a68f-de8a3a61b6f9.png)
it[it]
s[semple]
sample stops
![2022_12_19_128](https://user-images.githubusercontent.com/120786204/208516131-d5d07dd4-5a65-4a5b-a3a1-1f5b5c101545.png)
change settings:
stop again on one step
![asd (10)](https://user-images.githubusercontent.com/120786204/208516330-e39a926e-b0af-4239-875c-b6f86ee05687.png)
select previos cal:
![2022_12_19_123888](https://user-images.githubusercontent.com/120786204/208516377-921b949f-86dd-4cee-aa8f-41dfeef7290d.png)
change settings:
![2022_12_19_1288](https://user-images.githubusercontent.com/120786204/208516421-c40c886e-1b38-4066-aa21-a756c7859740.png)
well be A cal 2 mod a
new Cal[not awresume] update Letter _A_ on _B_ etc
so awresume update _Cal_1_ on _Cal_2_
but if have somes Cal1 Cal2 Cal3 etc but chosen some like Cal2 thay go on _Cal_3_ _mod_b_ cos _Cal_3_ _mod_a_ conflict
[A] [Cal 1] [mod a]
awresume [A] [Cal 1] [mod a]    be     [A] [Cal 2] [mod a]
awresume [A] [Cal 2] [mod a]    be     [A] [Cal 3] [mod a] 
new                                            be     [B] [Cal 1] [mod a] 
awresume [B] [Cal 1] [mod a]    be     [B] [Cal 2] [mod a] 
awresume [B] [Cal 2] [mod a]    be     [B] [Cal 3] [mod a] 
awresume [B] [Cal 1] [mod a]    be     [B] [Cal 2] [mod b] (set B_Cal_1 have Cal_2 [max mod a_a]conflict up mod_in_ b)
awresume [B] [Cal 2] [mod a]    be     [B] [Cal 3] [mod c] (set B_Cal_2 have Cal_3 [max mod a_b]conflict up mod_in_ c)
awresume [B] [Cal 2] [mod a]    be     [B] [Cal 3] [mod d] (set B_Cal_2 have Cal_3 [max mod a_c]conflict up mod_in_ d)
sets plased in (unworked/SesioName_Set_Cal_mod)
bases Set plased (worked/SesioName)
ckpts plased (worked/onreused/SesioName_someblabla)
when all sets erased the do with [bases Set]
SF upload all gen sample /id if change day-times
![asd (105)](https://user-images.githubusercontent.com/120786204/208516683-7734d09f-b5e3-4829-ac66-0d3085bd1cfb.png)

**Comment by symplijikes** [comment]
[lights.py.zip](https://github.com/TheLastBen/fast-stable-diffusion/files/10268001/lights.py.zip)
[sesion.py.zip](https://github.com/TheLastBen/fast-stable-diffusion/files/10268002/sesion.py.zip)
so fix erase base model nou they make def dir in previos place
so fix 2step model(en wps LBD)
so fix conflicts now be
![mv](https://user-images.githubusercontent.com/121100076/208657942-3ee799d0-d9c9-495f-ba2a-d9cc1bae7f45.jpg)

**Comment by pinguesscosa** [comment]
[lights.py.zip](https://github.com/TheLastBen/fast-stable-diffusion/files/10315774/lights.py.zip) [modeslights]
in pevios liths may be shift sed.. lines a so thet a fixed use this fil

**Comment by jaksyooo** [comment]
[MODESSIMPLESS] [simple.py.zip](https://github.com/TheLastBen/fast-stable-diffusion/files/10322888/simple.py.zip) 
lates explain fowhatse

**Comment by jakmiooo** [comment]
so... how a La Ben say big names no have a sense..... PT [a coool asgdfagsfdgfa nice machine]... i CPT [a coool machine]... dont needs, 
if PT well be use [a coool asgdfagsfdgfa nice machine] fo changeds token {a} {coool} {asgdfagsfdgfa} {nie} {machine},
CPT used only fo gen CLASS INSTANCE, all infos get a fom a INSTANCE, CLASS INSTANCE setuped in INSTANCE,
about {asgdfagsfdgfa},
in new [MODESSIMPLES] changeds token what a no in voces, all 3lettes,
if use big {asgdfagsfdgfa} exist tokens may be in {asgdfagsfdgfa} / {asgdfa------------} also contain infos PT / CPT
in changeds [MODESSESIONSS] [sesion.py.zip](https://github.com/TheLastBen/fast-stable-diffusion/files/10322909/sesion.py.zip) on steps gen new sed~~
i also about 3lettes tokens
if {zxo} contain nexts tokens
['z', 'zz', 'zzz', 'zzx', 'zzo', 'zx', 'zxz', 'zxx', 'zxo', 'zo', 'zoz', 'zox', 'zoo', 'x', 'xz', 'xzz', 'xzx', 'xzo', 'xx', 'xxz', 'xxx', 'xxo', 'xo', 'xoz', 'xox', 'xoo', 'o', 'oz', 'ozz', 'ozx', 'ozo', 'ox', 'oxz', 'oxx', 'oxo', 'oo', 'ooz', 'oox', 'ooo']
{asgdf} contain
['a', 'aa', 'aaa', ~80 lines , 'fffdf', 'fffff']
